src/main/scala/mesosphere/marathon/core/task/update/impl/steps/ScaleAppUpdateStepImpl.scala

```diff
@@ -22,7 +22,7 @@ class ScaleAppUpdateStepImpl @Inject() (
   private[this] lazy val schedulerActor = schedulerActorProvider.get()
 
   private[this] def scalingWorthy: Condition => Boolean = {
-    case Condition.Reserved | Condition.UnreachableInactive | _: Condition.Terminal => true
+    case Condition.UnreachableInactive | _: Condition.Terminal => true
```

Comment:
Is this true? We have `ReservedTerminal` and `Reserved`. I'm not sure what `ReservedTerminal` was used for.

Reply:
after phase 2 reservedterminal and reserved was the same see https://github.com/mesosphere/marathon/pull/6658/files#diff-4c766f1e11cd8f4e0f48c1a4392b2b41L43